src/nvim/po/fr.po

```diff
@@ -5264,12 +5264,21 @@ msgstr "Vim est un logiciel libre"
 msgid "Help poor children in Uganda!"
 msgstr "Aidez les enfants pauvres d'Ouganda !"
 
-msgid "type  :help iccf<Enter>       for information "
-msgstr "tapez  :help iccf<Entr�e>       pour plus d'informations          "
+msgid "type  :help nvim<Enter>       if you are new! "
+msgstr "tapez  :help nvim<Entr�e>       si vous �tes nouveau!             "
+
+msgid "type  :CheckHealth<Enter>     to optimize Nvim"
+msgstr "tapez  :CheckHealth<Entr�e>     pour optimiser Nvim               "
 
 msgid "type  :q<Enter>               to exit         "
 msgstr "tapez  :q<Entr�e>               pour sortir du programme          "
 
+msgid "type  :help<Enter>            for help        "
+msgstr "tapez  :help<Entr�e>            pour optenir de l'aide            "
```

Comment:
Thank you! Made a new PR to fix the typo. (https://github.com/neovim/neovim/pull/6094)